### PR TITLE
Update CSV import instructions

### PR DESCRIPTION
**Pull request by marselester**
Instructions were tested using the following setup.

Postgres 15.
```
$ docker run --name=pg --rm -it -v $(pwd)/csv:/csv -e POSTGRES_HOST_AUTH_METHOD=trust postgres:15-alpine
$ docker exec -it pg /bin/sh
# psql -U postgres
```

MySQL 8.2.0.
```
$ go run github.com/maxmind/geoip2-csv-converter -block-file GeoIP2-City-Blocks-IPv4.csv -include-hex-range -output-file GeoIP2-City-Blocks-IPv4-Hex.csv
$ go run github.com/maxmind/geoip2-csv-converter -block-file GeoIP2-City-Blocks-IPv6.csv -include-hex-range -output-file GeoIP2-City-Blocks-IPv6-Hex.csv
$ docker run --name=mysql --rm -it -v $(pwd)/csv:/var/lib/mysql-files/ -e MYSQL_ROOT_PASSWORD=my-secret-pw mysql:8.2.0
$ docker exec -it mysql /bin/sh
> mysql --password=my-secret-pw
```

```sql
create database test;
use test;
```

### CODE REVIEW

File: content/geoip/importing-databases/postgresql.mdx [hunk 1]
```diff
@@ -86,19 +86,18 @@ We can now import the contents of `GeoIP2-City-Blocks-IPv4.csv` and
...
-\copy geoip2_network(
-  network, geoname_id, registered_country_geoname_id, represented_country_geoname_id,
-  is_anonymous_proxy, is_satellite_provider, postal_code, latitude, longitude, accuracy_radius, is_anycast
-) from 'GeoIP2-City-Blocks-IPv6.csv' with (format csv, header);
+\copy geoip2_network from 'GeoIP2-City-Blocks-IPv6.csv' with (format csv, header);
 ```
 
+If there are errors such as `missing data for column` or
+`extra data after last expected column`, alter the table to meet the CSV file
+structure. For example, GeoLite2-City CSV file doesn't contain `is_anycast`, so
```
Comment (horgh) [inline content/geoip/importing-databases/postgresql.mdx:98]: GeoLite2-City is going to have that column actually. We could drop this last sentence I think.

**Review by horgh — CHANGES_REQUESTED** [review]

File: content/geoip/importing-databases/mysql.mdx [hunk 2]
```diff
@@ -173,14 +173,11 @@ limit 1;
 +------------+-------------------------------+--------------------------------+-------------+----------+-----------+-----------------+
```
[the remark this answers is not in the document]
Comment (marselester) [reply content/geoip/importing-databases/mysql.mdx:173]: Yes, they did.

<details>

```sql
create table geoip2_network (
  network_start varbinary(16) not null,
  network_end varbinary(16) not null,
  geoname_id int,
  registered_country_geoname_id int,
  represented_country_geoname_id int,
  is_anonymous_proxy bool,
  is_satellite_provider bool,
  postal_code text,
  latitude float,
  longitude float,
  accuracy_radius int,
  index(network_start),
  index(network_end)
);

-- Load CSV file with extra column (is_anycast).
load data infile '/var/lib/mysql-files/GeoIP2-City-CSV_Example/GeoIP2-City-Blocks-IPv6-Hex.csv'
into table geoip2_network
fields terminated by ',' enclosed by '"' lines terminated by '\n' ignore 1 rows
(@network_start, @network_end, @geoname_id, @registered_country_geoname_id, @represented_country_geoname_id,
 @is_anonymous_proxy, @is_satellite_provider, @postal_code, @latitude, @longitude, @accuracy_radius, @is_anycast)
set network_start = unhex(@network_start),
    network_end = unhex(@network_end),
    geoname_id = nullif(@geoname_id, ''),
    registered_country_geoname_id = nullif(@registered_country_geoname_id, ''),
    represented_country_geoname_id = nullif(@represented_country_geoname_id, ''),
    is_anonymous_proxy = nullif(@is_anonymous_proxy, ''),
    is_satellite_provider = nullif(@is_satellite_provider, ''),
    postal_code = nullif(@postal_code, ''),
    latitude = nullif(@latitude, ''),
    longitude = nullif(@longitude, ''),
    accuracy_radius = nullif(@accuracy_radius, '');

Query OK, 230 rows affected (0.02 sec)
Records: 230  Deleted: 0  Skipped: 0  Warnings: 0
```

</details>

File: content/geoip/importing-databases/mysql.mdx [hunk 3]
```diff
@@ -173,14 +173,11 @@ limit 1;
...
 | geoname_id | registered_country_geoname_id | represented_country_geoname_id | postal_code | latitude | longitude | accuracy_radius |
 +------------+-------------------------------+--------------------------------+-------------+----------+-----------+-----------------+
-|    4930956 |                       6252001 |                           NULL | 02118       |  42.3388 |  -71.0726 |             100 |
+|    4952487 |                       6252001 |                           NULL | 02777       |  41.7505 |  -71.2089 |             100 |
 +------------+-------------------------------+--------------------------------+-------------+----------+-----------+-----------------+
-1 row in set (6.16 sec)
+1 row in set (0.03 sec)
 ```
 
-While that yields the correct results, we notice that the query performed so
```
[the remark this answers is not in the document]
Comment (marselester) [reply content/geoip/importing-databases/mysql.mdx:181]: > I wonder why the original example performed much worse.

I don't know really, I am not familiar with mysql. The table is created with indexes on `network_start` and `network_end`, so 30ms query time is on par with postgres (7 ms with GiST index).

I tested with the full data set (the example data set doesn't have info about 146.243.121.22 address).

<details>

```sql
mysql> create database test;
Query OK, 1 row affected (0.01 sec)

mysql> use test;
Database changed
mysql> create table geoip2_network (
    ->   network_start varbinary(16) not null,
    ->   network_end varbinary(16) not null,
    ->   geoname_id int,
    ->   registered_country_geoname_id int,
    ->   represented_country_geoname_id int,
    ->   is_anonymous_proxy bool,
    ->   is_satellite_provider bool,
    ->   postal_code text,
    ->   latitude float,
    ->   longitude float,
    ->   accuracy_radius int,
    ->   is_anycast bool,
    ->   index(network_start),
    ->   index(network_end)
    -> );
Query OK, 0 rows affected (0.03 sec)

mysql> load data infile '/var/lib/mysql-files/GeoIP2-City-Blocks-IPv6-Hex.csv'
    -> into table geoip2_network
    -> fields terminated by ',' enclosed by '"' lines terminated by '\n' ignore 1 rows
    -> (@network_start, @network_end, @geoname_id, @registered_country_geoname_id, @represented_country_geoname_id,
    ->  @is_anonymous_proxy, @is_satellite_provider, @postal_code, @latitude, @longitude, @accuracy_radius, @is_anycast)
    -> set network_start = unhex(@network_start),
    ->     network_end = unhex(@network_end),
    ->     geoname_id = nullif(@geoname_id, ''),
    ->     registered_country_geoname_id = nullif(@registered_country_geoname_id, ''),
    ->     represented_country_geoname_id = nullif(@represented_country_geoname_id, ''),
    ->     is_anonymous_proxy = nullif(@is_anonymous_proxy, ''),
    ->     is_satellite_provider = nullif(@is_satellite_provider, ''),
    ->     postal_code = nullif(@postal_code, ''),
    ->     latitude = nullif(@latitude, ''),
    ->     longitude = nullif(@longitude, ''),
    ->     accuracy_radius = nullif(@accuracy_radius, ''),
    ->     is_anycast = nullif(@is_anycast, '');
Query OK, 2341927 rows affected (13.49 sec)
Records: 2341927  Deleted: 0  Skipped: 0  Warnings: 0

mysql> load data infile '/var/lib/mysql-files/GeoIP2-City-Blocks-IPv4-Hex.csv'
    -> into table geoip2_network
    -> fields terminated by ',' enclosed by '"' lines terminated by '\n' ignore 1 rows
    -> (@network_start, @network_end, @geoname_id, @registered_country_geoname_id, @represented_country_geoname_id,
    ->  @is_anonymous_proxy, @is_satellite_provider, @postal_code, @latitude, @longitude, @accuracy_radius, @is_anycast)
    -> set network_start = unhex(@network_start),
    ->     network_end = unhex(@network_end),
    ->     geoname_id = nullif(@geoname_id, ''),
    ->     registered_country_geoname_id = nullif(@registered_country_geoname_id, ''),
    ->     represented_country_geoname_id = nullif(@represented_country_geoname_id, ''),
    ->     is_anonymous_proxy = nullif(@is_anonymous_proxy, ''),
    ->     is_satellite_provider = nullif(@is_satellite_provider, ''),
    ->     postal_code = nullif(@postal_code, ''),
    ->     latitude = nullif(@latitude, ''),
    ->     longitude = nullif(@longitude, ''),
    ->     accuracy_radius = nullif(@accuracy_radius, ''),
    ->     is_anycast = nullif(@is_anycast, '');
Query OK, 11397019 rows affected (50.58 sec)
Records: 11397019  Deleted: 0  Skipped: 0  Warnings: 0

mysql> select geoname_id, registered_country_geoname_id, represented_country_geoname_id,
    ->        postal_code, latitude, longitude, accuracy_radius
    -> from geoip2_network
    -> where inet6_aton('146.243.121.22') between network_start and network_end
    -> limit 1;
+------------+-------------------------------+--------------------------------+-------------+----------+-----------+-----------------+
| geoname_id | registered_country_geoname_id | represented_country_geoname_id | postal_code | latitude | longitude | accuracy_radius |
+------------+-------------------------------+--------------------------------+-------------+----------+-----------+-----------------+
|    4952487 |                       6252001 |                           NULL | 02777       |  41.7505 |  -71.2089 |             100 |
+------------+-------------------------------+--------------------------------+-------------+----------+-----------+-----------------+
1 row in set (0.04 sec)
```

</details>

**Review by horgh — APPROVED** [review]

**Review by christophermluna — APPROVED** [review]
this looks great, thanks for all y'all's work on this

**Pull request merged**